pallets/elections/src/impls.rs

```diff
@@ -0,0 +1,343 @@
+use crate::{Config, ErasReserved, MembersPerSession, Pallet, SessionValidatorBlockCount};
+use frame_election_provider_support::sp_arithmetic::Perquintill;
+use frame_support::{pallet_prelude::Get, traits::Currency};
+use primitives::TOKEN;
+use sp_staking::{EraIndex, SessionIndex};
+use sp_std::vec::Vec;
+
+fn scale_total_exposure(total: u128) -> u128 {
+    // to cover minimal possible theoretical stake (25k) and avoid loss of accuracy we need to scale
+    100 * total / TOKEN
```

Comment:
This is not needed now? As we don't count points per block, but per session

Reply:
It is as you said, removed